drake/examples/HolonomicDrive/HolonomicDrivePlant.m

```diff
@@ -0,0 +1,246 @@
+classdef HolonomicDrivePlant < SecondOrderSystem
+  properties (SetAccess = private, GetAccess = public)
+    m
+    I
+    wheels
+  end
+
+  methods
+    function obj = HolonomicDrivePlant(wheels, I, m)
+      % wheels: struct of
+      %   pos:      offset from center of mass
+      %   r:        wheel radius
+      %   driveDir: drive direction, a unit vector
+      %   slipDir:  slip direction. For mecanum wheels, this is not
+      %             orthogonal to the drive direction
+      %   omegaMax: motor free-running speed
+      %   tauMax:   motor stall-torque
+      % I: moment of inertia of entire robot about vertical axis
+      % m: total mass of robot

```

Comment:
What does that buy over a struct? Passing these variables as unnamed constructor arguments to `HolonomicDrivePlantWheel` would be confusing, at which point the constructor becomes an alias for `struct`.

That will currently cause `symbolify` to fail also, as I didn't implement symbolizing classes.